### PR TITLE
Iavp update

### PR DESCRIPTION
**Pull request by Jrad7**

### CODE REVIEW

File: pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm [hunk 1]
```diff
@@ -106,14 +106,13 @@ sub generate_advice {
 
 sub _get_purchase_and_install_template {
     return << 'TEMPLATE';
-[%- locale.maketext('[asis,Imunify360] delivers sophisticated detection and display of security threats, powered by a self-learning firewall with herd immunity. It blocks attacks in real-time using a combination of technologies, including:') %]
+[%- locale.maketext('Use [asis,Imunify360] for a comprehensive suite of protection against attacks on your servers.') %]
```
Comment (teodesian) [inline pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm]: Ouch, we have to update strings in cP/WHM to make this work?  Makes no sense whatsoever this is in a separate repo.

Comment (bdraco) [reply pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm]: Do we have an automated way to pull in these string updates?

Comment (teodesian) [reply pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm]: Not to my knowledge.  cplint does not run lt checking on submodules.

Comment (bdraco) [reply pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm]: Thanks.  Tracking as CPANEL-31966

**Pull request merged**